src/integrations/GA4/browser.js

```diff
@@ -51,9 +50,7 @@ export default class GA4 {
     if (this.sendUserId) {
       gtagParameterObject.user_id = userId;
     }
-    if (this.debugMode) {
-      gtagParameterObject.debug_mode = true;
-    }
+    gtagParameterObject.debug_mode = true;
```

Comment:
Is it correct to made debugMode by default true in production as well? I can see [here](https://support.google.com/analytics/answer/7201382?hl=en&utm_id=ad#:~:text=It%27s%20important%20to%20filter%20out%20traffic%20while%20debug%20mode%20is%20on%20so%20the%20data%20you%20collect%20doesn%27t%20negatively%20impact%20the%20data%20in%20your%20reports.%20You%20can%20filter%20out%20developer%20traffic%20so%20the%20data%20you%20collect%20when%20debug%20mode%20is%20enabled%20doesn%27t%20impact%20your%20report%20data.%20Learn%20more) google mentioned to filter out test data and report data.

Reply:
@ItsSudip enabling debug_view won't cause any issue, in order to filter test data [this](https://support.google.com/analytics/answer/10104470?hl=en) steps needs to be followed

Reply:
Yeah as they asking to filter out the test data doesn't it mean enabling debug mode during testing only?

Reply:
This will affect [GA4 report data](https://support.google.com/analytics/answer/7201382?hl=en#zippy=%2Cgoogle-tag-gtagjs)?